app/components/token-badge/token-badge.tsx

```diff
@@ -6,7 +6,9 @@ import { Badge } from "../badge";
 export function TokenBadge({ token, quantity }: { token: Token; quantity?: string }) {
   return (
     <Badge>
-      {quantity ? <span className="text-xs"> {fromTokenAmount(quantity, token.decimals)}</span> : null}
+      {quantity ? (
+        <span className="text-xs mt-0.5 mr-1"> {fromTokenAmount(quantity, token?.decimals ?? 18)}</span>
```

Comment:
This might seem nit picky but I actually think it should be a space between the two and not a margin. For copy+paste sake and formatting.